cypress.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
```

Comment:
Can we please add a TODO comment here indicating the ticket number for this task?

Reply:
@DorianKwan is this tied to a ticket you have already written?

Reply:
We already have a `cypress.config.ts` in the root of the `/test` dir/project.  What is this one used for?

Reply:
Duplicate has been removed!